includes/wikia/api/LogEventsApiController.class.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * API which allows you to add items to Special:Log
+ *
+ * This is basically HTTP interface to the these PHP methods:
+ *
+ * LogPage::__construct ($type, $rc=true, $udp= 'skipUDP')
+ * LogPage::addEntry ($action, $target, $comment, $params=array(), $doer=null)
+ *
+ * @see https://www.mediawiki.org/wiki/Manual:LogPage.php
+ * @see https://www.mediawiki.org/wiki/Manual:Log_actions
+ */
+
+class LogEventsApiController extends WikiaApiController {
+
+	/**
+	 * Add a new entry to logging table
+	 *
+	 * @throws BadRequestException
+	 */
+	public function add() {
+		global $wgTheSchwartzSecretToken, $wgTitle, $wgUser;
+
+		if ( !$this->request->wasPosted() ) {
+			throw new BadRequestException( 'This request must be POSTed' );
+		}
+
+		if ( !Wikia\Security\Utils::matchToken( $wgTheSchwartzSecretToken, $this->request->getVal( 'token' ) ) ) {
```

Comment:
@michalroszka, @ArturKlajnerok do you know of any reason why we decided to use `'secret'` for the [helios end points](https://github.com/Wikia/app/blob/dev/extensions/wikia/Helios/HelperController.class.php#L221) instead of `token`? Can we consolidate on one?